source/lovelace/header-footer.markdown

```diff
@@ -71,6 +71,11 @@ entities:
       required: false
       description: Override the entity icon. You can use any icon from [MaterialDesignIcons.com](http://materialdesignicons.com). Prefix the icon name with `mdi:`, ie `mdi:home`.
       type: string
+    show_icon:
```

Comment:
Seems like there is much more here:

![image](https://user-images.githubusercontent.com/195327/113697040-fe60fe00-96d2-11eb-9e69-fc78e599aa57.png)


`show_name`, `state_color`, `hold_action`, `tap_action`.

Additionally, I see this in the code:

![image](https://user-images.githubusercontent.com/195327/113697250-3e27e580-96d3-11eb-8945-d97f8dda3b85.png)


However `double_tap_action` doesn't seems to be defined.

CC @bramkragten ?